pkg/route/admission/admission_test.go

```diff
@@ -0,0 +1,180 @@
+package admission
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/admission"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/cache"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	routeapi "github.com/openshift/origin/pkg/route/api"
+	"strings"
+	"testing"
+)
+
+func TestAdmission(t *testing.T) {
```

Comment:
When you update the Handles implementation, add a test that you only handle what you say you handle.